### PR TITLE
Hebrew language support

### PR DESCRIPTION
**Pull request by mosheDO**

### CODE REVIEW

**Comment by mosheDO** [comment]
finish all hebrew translation

File: youtube_dl_gui/optionsframe.py [hunk 1]
```diff
@@ -273,7 +273,8 @@ class GeneralTab(TabPanel):
         ('ko_KR', 'Korean'),
         ('pt_BR', 'Portuguese'),
         ('ru_RU', 'Russian'),
-        ('es_ES', 'Spanish')
+        ('es_ES', 'Spanish'),
+		('he-IL', 'Hebrew')
```
Comment (MrS0m30n3) [inline youtube_dl_gui/optionsframe.py]: * Sort alphabetically as the guide says
* Change the language code from 'he-IL' -> 'he_IL'
* Rename the 'he-IL' directory under the **locale** dir to 'he_IL' in order to match the new language code
* Follow the Python's conventions and align the Hebrew language to the other items (4-spaces)

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 2]
```diff
@@ -0,0 +1,588 @@
+# Youtube-dlG localization file.
+# FIRST AUTHOR: Sotiris Papadopoulos <ytubedlg@gmail.com>, 2015.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: youtube-dlg 0.4\n"
+"POT-Creation-Date: 2017-06-15 17:13+EEST\n"
+"PO-Revision-Date: 2017-06-15 17:14+EEST\n"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: Update the revision date

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 3]
```diff
@@ -0,0 +1,588 @@
...
+"Last-Translator: Sotiris Papadopoulos <ytubedlg@gmail.com>\n"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: Update the last translator field with valid contact details

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 4]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Successfully downloaded {0} URL(s) in {1} day(s) {2} hour(s) {3} minute(s) {4} second(s)"
+msgstr "הוריד בהצלחה את {0} כתובות האתרים תוך {1} ימים {2} שעות {3} דקות {4} שניות"
+
+#: youtube_dl_gui/mainframe.py:119
+msgid "Downloads completed"
+msgstr "ההורדה הושלמה"
+
+#: youtube_dl_gui/mainframe.py:120
+msgid "Total Progress: {0:.1f}% | Queued ({1}) Paused ({2}) Active ({3}) Completed ({4}) Error ({5})"
+msgstr "סה"כ התקדמות: {0:.1f}% | בהמתנה ({1}) בהפסקה ({2}) פעילים ({3}) הושלמו ({4}) בשגיאה ({5})"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: Please if you have to use quotes in your translations use single quotes ( ' ) and not double quotes ( " )

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 5]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Get command"
+msgstr "קבל פקודה"
+
+#: youtube_dl_gui/mainframe.py:237
+msgid "Open destination"
+msgstr "פתח יעד"
+
+#: youtube_dl_gui/mainframe.py:238
+msgid "Re-enter"
+msgstr "הזן מחדש"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: That translates to "Please re-enter". You should remove the "Please" part

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 6]
```diff
@@ -0,0 +1,588 @@
...
+
+#: youtube_dl_gui/mainframe.py:458
+msgid "Resume"
+msgstr "התחל מחדש"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: Google translates this to "Start over" which is like the opposite of "Resume" (continue from were you stopped).
Are you sure this is correct?

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 7]
```diff
@@ -0,0 +1,588 @@
...
+msgid "No items selected. Please pick an action"
+msgstr "לא נבחרו פריטים. נא בחר פעולה"
+
+#: youtube_dl_gui/mainframe.py:516
+msgid "Remove all"
+msgstr "מחק הכל"
+
+#: youtube_dl_gui/mainframe.py:516
+msgid "Remove completed"
+msgstr ""
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: Why this is empty?

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 8]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Are you sure you want to exit?"
+msgstr "אתה בטוח שאתה רוצה לצאת?"
+
+#: youtube_dl_gui/mainframe.py:1060
+msgid "Exit"
+msgstr "יציאה"
+
+#: youtube_dl_gui/mainframe.py:1301 youtube_dl_gui/mainframe.py:1451
+msgid "Cancel"
+msgstr "בטל"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: This translates to *void, null, idle* which have nothing in common with the *msgid* field.
Are you sure this is correct?

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 9]
```diff
@@ -0,0 +1,588 @@
...
+
+#: youtube_dl_gui/mainframe.py:1450
+msgid "OK"
+msgstr "אשר"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: This translates to *which*, which again is not equal to the *msgid* field.
Are you sure this is correct?

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 10]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Extra"
+msgstr "תוספות"
+
+#: youtube_dl_gui/optionsframe.py:315
+msgid "Language"
+msgstr "שפה"
+
+#: youtube_dl_gui/optionsframe.py:318
+msgid "Filename format"
+msgstr "פורמט שם הקובץ"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: According to Google the correct translation here it's: פורמט בשם הקובץ

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 11]
```diff
@@ -0,0 +1,588 @@
...
+msgid "low"
+msgstr "נמוך"
+
+#: youtube_dl_gui/optionsframe.py:468
+msgid "mid"
+msgstr "רגיל"
+
+#: youtube_dl_gui/optionsframe.py:473
+msgid "Video formats"
+msgstr "פורמטי וידאו"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: This translates to *Formats formats*, which makes no sense.
Are you sure this is correct?

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 12]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Restrict filenames to ASCII"
+msgstr "הגבל שמות קבצים ל- ASCII"
+
+#: youtube_dl_gui/optionsframe.py:326
+msgid "More options"
+msgstr "אפשרויות נוספות"
+
+#: youtube_dl_gui/optionsframe.py:327
+msgid "Confirm on exit"
+msgstr "אשר ביציאה"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: This translates to *Which is at the exit*, which again it's probably wrong.

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po [hunk 13]
```diff
@@ -0,0 +1,588 @@
...
+msgid "Proxy"
+msgstr "פרוקסי"
+
+#: youtube_dl_gui/optionsframe.py:744
+msgid "User agent"
+msgstr ""
+
+#: youtube_dl_gui/optionsframe.py:746
+msgid "Referer"
+msgstr "מפנה"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: https://en.wikipedia.org/wiki/HTTP_referer
Your translation makes no sense

**Review by MrS0m30n3 — COMMENTED** [review]
I used Google translate to check your PO file, if i'm wrong at any point please let me know.

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po (hunk 4 above)
Comment (mosheDO) [reply youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: change to single qoutes

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po (hunk 5 above)
Comment (mosheDO) [reply youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: I think you already removed it.

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po (hunk 6 above)
Comment (mosheDO) [reply youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: I change this in the fix commit

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po (hunk 7 above)
Comment (mosheDO) [reply youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: I added this

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po (hunk 8 above)
Comment (mosheDO) [reply youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: This is for sure the translation yoy can check at
https://translate.google.com/#en/iw/cancel
As you can see it translate to לבטל but בטל  is also

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po (hunk 9 above)
Comment (mosheDO) [reply youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: Yes i'm sure

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po (hunk 10 above)
Comment (mosheDO) [reply youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: I'm not sure about it, i need to check this again

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po (hunk 12 above)
Comment (mosheDO) [reply youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: https://translate.google.com/#en/iw/Confirm%20on%20exit

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po (hunk 11 above)
Comment (mosheDO) [reply youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: https://translate.google.com/#iw/en/%D7%A4%D7%95%D7%A8%D7%9E%D7%98%D7%99%20%D7%95%D7%99%D7%93%D7%90%D7%95

File: youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po (hunk 13 above)
Comment (mosheDO) [reply youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po]: https://he.wikipedia.org/wiki/Referer
As you can see in the first line on the hebrew wiki 
"ה-Referer (או דף **מפנה**) הוא כתובת ה-URL של דף האינטרנט הקודם שבו לחצו על קישור"

**Comment by MrS0m30n3** [comment]
@mosheDO are you still interested on this?

**Comment by a5r0n** [comment]
@MrS0m30n3  
I'm interested in it, what should be done to finish it?

**Pull request closed without merging**